Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -481,6 +481,12 @@ private string FindMGCB()
 
         private void DoBuild(string commands)
         {
+            if(!Environment.Is64BitOperatingSystem)
+            {
+                View.OutputAppend("Build failed:  Content can only be compiled from a 64 bit OS.");
```

Comment:
This blocks the compilation, not the Pipeline Tool.

That is when you press build, if your system is 32 bit, you will get that message.